tools/arxiv/scripts/pdf_scanner_fixed.py

```
@@ -0,0 +1,92 @@
+    print(f"Sample saved to: {output_file}")
+    
+    return sample_pdfs
+
+if __name__ == "__main__":
+    random.seed(42)  # For reproducible sampling
+    sample = quick_pdf_sample()
+    
+    # Show some sample papers
+    print("\nSample papers:")
+    for i, pdf in enumerate(sample[:10]):
+        print(f"  {i+1}. {pdf['arxiv_id']} ({pdf['year_month']}) - {pdf['size']/1024/1024:.1f} MB")
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)